blog/_posts/2018-09-17-sectumsempra-rgsoc.md

```diff
@@ -0,0 +1,138 @@
+---
+title: “How lucky are we to have something that makes saying goodbye so hard!”
+layout: post
+created_at: Mon Sep 17 2018
+permalink: blog/2018-09-17-team-sectumsempra
```

Comment:
Hi @avneet14027 ! Thanks for the PR. Amazing blog post and quiet very well written with the mention of personal experiences. 

Minor comment here. Permalink and and file name should be the same. Please fix that.

Reply:
Thanks @nerdyvaishali ! We enjoyed writing the blog post. Straight from the heart ❤️ 
I fixed the file name and the added the image credits.

Regards,
Avneet